src/fs.ts

```diff
@@ -11,8 +12,9 @@ export function getPackageJSON(cwd = process.cwd()): any {
       return data
     }
     catch (e) {
-      console.warn('Failed to parse package.json')
-      process.exit(0)
+      !programmatic ? console.warn('Failed to parse package.json') : 0
+      !programmatic ? process.exit(0) : 0
```

Comment:
Is was wondering if `exit 0` was intentional here?

Reply:
Yeah probably not. 

Can we do an `if` instead of ternary conditions?

Reply:
Sure, I also fixed some other linting issues and adding some tests as well.

Reply:
(Also changed exit code)